### DISCUSSION TITLE
error loading sirius-web

### DISCUSSION THREAD

**Discussion by AsmaIrfan12** [eclipse-sirius/sirius-components#4123] [Q&A]
after building the tool I am having problem running the java code....please if anyone can solve the below issue

$ java -jar sirius-web-application.jar\--spring.datasource.url=jdbc:postgresql://localhost:5433/sirius-web-db
    --spring.datasource.username=dbuser           --spring.datasource.password=dbpwd           --spring.liquibase.change-log=classpath:db/changelog/sirius-web.db.changelog.xml
Error: Unable to access jarfile sirius-web-application.jar--spring.datasource.url=jdbc:postgresql://localhost:5433/sirius-web-db

**Comment by frouene** [comment]
Hello @nabira12,

I think a space is missing between `sirius-web-application.jar` and `--spring.datasource.url`, because it's looking for a jar named `sirius-web-application.jar--spring.datasource.url=jdbc:postgresql://localhost:5433/sirius-web-db`, that does not exist.
